model_utxos.go

```diff
@@ -235,6 +235,25 @@ func newUtxoFromTxID(txID string, index uint32, opts ...ModelOps) *Utxo {
 	}
 }
 
+// getUtxos will get all the utxos with the given conditions
+func getUtxos(ctx context.Context, metadata *Metadata, conditions *map[string]interface{},
+	queryParams *datastore.QueryParams, opts ...ModelOps) ([]*Utxo, error) {
+
+	modelItems := make([]*Utxo, 0)
+	if err := getModelsByConditions(ctx, ModelUtxo, &modelItems, metadata, conditions, queryParams, opts); err != nil {
+		return nil, err
```

Comment:
Does opts need the `...`

`opts` vs `opts...`

Not sure if that is an error or not...